examples/constraints.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from scipy.optimize import rosen, rosen_der
+from cyipopt import minimize_ipopt
+
+x0 = np.array([0.5, 0])
+
+bounds = [np.array([0, 1]), np.array([-0.5, 2.0])]
+
+
+eq_cons = {'type': 'eq',
+           'fun' : lambda x: np.array([2*x[0] + x[1] - 1, x[0]**2 - 0.1])
+          }
+
+res = minimize_ipopt(rosen, x0, jac=rosen_der, bounds=bounds, constraints=[eq_cons])
+
+print(res)
```

Comment:
It would be ideal to add a unit test that checks this example, or a similar one.

Reply:
Agreed. See [here](https://github.com/mechmotum/cyipopt/blob/master/cyipopt/tests/unit/test_scipy_optional.py) for how we've written other tests that an scipy-optional. The test can simply be a copy (with the small required changes) of any of the three tests in that module currently marked with `@pytest.mark.skipif("scipy" not in sys.module)`.